### ISSUE TITLE
NTFS fail: check_function_parameters_quantity: FATAL: Function requires 4 parameters, but 3 is given

### ISSUE THREAD

**Issue by stevage** [WoeUSB/WoeUSB#90]
## Describe the bug ##

Failure after formatting drive as NTFS.

```
$ umount /dev/sdc1; sudo bash woeusb-5.2.0.bash --tgt-fs NTFS --device win10.iso /dev/sdc
umount: /dev/sdc1: not mounted.
WoeUSB v5.2.0
==============================
Info: Mounting source filesystem...
Info: Wiping all existing partition table and filesystem signatures in /dev/sdc...
/dev/sdc: 2 bytes were erased at offset 0x000001fe (dos): 55 aa
/dev/sdc: calling ioctl to re-read partition table: Success
Info: Ensure that /dev/sdc is really wiped...
Info: Creating new partition table on /dev/sdc...
Info: Creating target partition...
Info: Making system realize that partition table has changed...
Info: Wait 3 seconds for block device nodes to populate...
Cluster size has been automatically set to 4096 bytes.
Creating NTFS volume structures.
mkntfs completed successfully. Have a nice day.
check_function_parameters_quantity: FATAL: Function requires 4 parameters, but 3 is given
Info: Unmounting and removing "/tmp/woeusb-source-20211110-230125-Wednesday.WrhXLd"...
Info: Unmounting and removing "/tmp/woeusb-target-20211110-230125-Wednesday.pmbTJU"...
umount: /tmp/woeusb-target-20211110-230125-Wednesday.pmbTJU: not mounted.
Warning: Unable to unmount "/tmp/woeusb-target-20211110-230125-Wednesday.pmbTJU".
Warning: Some mountpoints are not unmount/cleaned successfully and must be done manually
Info: You may now safely detach the target device
```

## To Reproduce ##

What I did:

1. Ran Ubuntu 20.04.3 from a bootable UBS. 
2. Installed dependencies and ran the above command on a 16GB USB key.

## Environment ##

- OS: Ubuntu 20.04.3
- Bash: [e.g. `bash --version`]
- WoeUSB: 5.2.0

## Additional

Repeating the process with FAT instead of NTFS gets past that point.

**Comment by sudosuse** [comment]
@stevage I tried a lot and to fix it and it didnt work, but i found a workaround to get past that nasty bug:
1: You need to download an older verision (it works with 5.1.3)
2:then put the files permission (chmod +x)
3:redo the command you did but instead of woeusb-5.2.0, you type woeusb-5.1.3.bash.

Now, normally it should work, so i hope it works for you. :) Bye !

**Comment by brlin-tw** [comment]
Apologies for the ignorance, I am aware of this issue and will work on it ASAP.

**Comment by sudosuse** [comment]
Cool !, but @brlin-tw  that a PHP error, right ? how a error like that could happen ?

**Comment by brlin-tw** [comment]
@sudosuse > how an error like that could happen

To prevent errors that caused by adding/removing parameters to the function while not simultaneously fixing all of its callers a special utility function is implemented to check the function argument count in the runtime, the goal for it is to quickly identify such errors and fixing during the QA stage, which unfortunately did not happen before the 5.2.0 release.

**Comment by brlin-tw** [comment]
This problem is fixed in bdd3466ee30c631a97c8e536a51080d3e372b0f3 revision.

**Comment by brlin-tw** [comment]
The fix to this problem is released in [WoeUSB 5.2.1](https://github.com/WoeUSB/WoeUSB/releases/tag/v5.2.1).
